crates/sncast/src/main.rs

```diff
@@ -174,6 +177,7 @@ async fn run_async_command(
                 declare.nonce,
                 build_config,
                 wait_config,
+                false,
```

Comment:
shouldn't this also use script_ui.quiet?

Reply:
This one is executed via `sncast declare`, not from `script`

Reply:
![obraz](https://github.com/foundry-rs/starknet-foundry/assets/9470712/47eac345-17e1-4ce6-b0fb-5ca54be21276)